ansible/roles/nginx/vars/motech.yml

```diff
@@ -20,3 +20,6 @@ nginx_sites:
      - name: /supply-dhis
        proxy_pass: http://motech-main.internal.commcarehq.org:8888/supply-dhis
        proxy_read_timeout: 180s
+     - name: /cdc-moz
+       proxy_pass: http://motech3.internal.commcarehq.org:8080/cdc-moz
```

Comment:
@sheelio I think this config is incorrect. If this is on virginia it's `internal-va`